static/templates/dialog/base-dialog.hbs

```diff
@@ -27,7 +27,7 @@
       <label>{{localize "DIALOG.DialogModifiers"}}</label>
       <ul class="modifier-list">
         {{#each data.scripts}}
-            <li class="modifier {{#if this.isActive}}active {{else if this.isHidden}}hidden{{/if}}" data-tooltip="{{this.effect.name}} {{#if this.options.targeter}}(Target){{/if}}" data-tooltip-direction="RIGHT" data-index="{{@key}}" >&#8226; <a>{{this.Label}}</a></li>
+            <li class="modifier {{#if this.isActive}}active {{else if this.isHidden}}hidden{{/if}}" data-tooltip="{{this.effect.name}} {{#if this.options.targeter}}({{localize 'Target'}}){{/if}}" data-tooltip-direction="RIGHT" data-index="{{@key}}" >&#8226; <a>{{this.Label}}</a></li>
         {{/each}}
       </ul>
     </div>
```

Comment:
This should probably just use the partial provided by the warhammer library (which needs to be localized too)

Reply:
it seams that none of the partial templates are being used in WFRP4e system so far. as of Lib translation, I might need to prepare PR there anyway.